vermouth/ffinput.py

```diff
@@ -180,9 +181,12 @@ def finalize_section(self, previous_section, ended_section):
             self.current_modification.citations.update(self.citations)
             self.force_field.modifications[self.current_modification.name] = self.current_modification
 
-    def get_context(self, context_type):
+    def get_context(self, context_type=''):
+        if not context_type:
+            context_type = self.section[0]
```

Comment:
Do we really need this? As this is never called in the tests I'm not sure it makes any sense to set this default. You can just add the context type explicitly to the decorator right? If we keep it I think the context type should also be removed from the other decorators where it is not needed.